src/test/ui/anonymous-higher-ranked-lifetime.stderr

```diff
@@ -154,4 +154,5 @@ LL | fn h2<F>(_: F) where F: for<'t0> Fn(&(), Box<Fn(&())>, &'t0 (), fn(&(), &()
 
 error: aborting due to 11 previous errors
 
-If you want more information on this error, try using "rustc --explain E0631"
+error: For more information about this error, try `rustc --explain E0631`.
```

Comment:
It's [still the case that](https://github.com/rust-lang/rust/pull/48337#issuecomment-369670586) the relative ordering of the `--explain` usage and "aborting due to ..." messages should arguably be reverse of this.